proxy/src/coco/source.rs

```diff
@@ -217,9 +235,43 @@ pub fn blob(
         .last_commit(commit_path)?
         .map(|c| CommitHeader::from(&c));
     let (_rest, last) = p.split_last();
-    let content = match std::str::from_utf8(&file.contents) {
-        Ok(content) => BlobContent::Ascii(content.to_string()),
-        Err(_) => BlobContent::Binary,
+    let content = match (std::str::from_utf8(&file.contents), theme) {
+        (Ok(content), None) => BlobContent::Ascii(content.to_owned()),
+        (Ok(content), Some(theme)) => {
+            use syntect::easy::HighlightLines;
+            use syntect::highlighting::ThemeSet;
+            use syntect::util::LinesWithEndings;
```

Comment:
Let's avoid inline `use` statements, we don't use them anywhere else in the codebase. Normally clippy should complain. :thinking: